Sezai/index.js

```diff
@@ -0,0 +1,20 @@
+// 1.
+
+let str1 = "Ghassanovic";
+console.log(str1);
+
+//2.
+console.log("access 2nd and 6th elements ==> ", str1.slice(2, 7));
```

Comment:
You just need to get access to the 2nd element and 6th element.
console.log(str1[1], str1[5]);